server/service/src/sync/test/integration/omsupply_central/pack_variant.rs

```diff
@@ -0,0 +1,108 @@
+use crate::{
+    sync::{
+        test::integration::{
+            central_server_configurations::NewSiteProperties, GraphqlRequest, SyncRecordTester,
+            TestStepData,
+        },
+        translations::IntegrationOperation,
+    },
+    u32_to_i32,
+};
+
+use rand::Rng;
+use repository::PackVariantRow;
+use serde_json::json;
+use util::uuid::uuid;
+
+pub(crate) struct PackVariantTester;
+
+impl SyncRecordTester for PackVariantTester {
+    fn test_step_data(&self, site_properties: &NewSiteProperties) -> Vec<TestStepData> {
+        let mut result = Vec::new();
+
+        let item_id = uuid();
+        // STEP 1 - insert
+        let item_json = json!({
+            "ID": item_id.clone(),
+            "type_of": "general",
+
+        });
+        let mut rng = rand::thread_rng();
+        let pack_size: u32 = rng.gen();
```

Comment:
Any special reason why you use a random pack_size?

Reply:
I think that was a confusion on my part (api enforces uniqueness of pack_size/item, but we creating new item anyway), changed: 1e0109286f154fdb9e7363062519c6e209794e6a